code/datums/dog_fashion.dm

```diff
@@ -200,3 +200,7 @@
 	D.mutations.Add(BREATHLESS)
 	D.atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
 	D.minbodytemp = 0
+
+/datum/dog_fashion/head/fried_vox_empty
+	name = "REAL_NAME, the number one fan of KFV"
```

Comment:
Having Ian called

`Ian, the number one fan of KFV` is not only a super long name, but also seems way out of place. Thats more of a description, not a mob name.

Reply:
what about "#1 KFV fan Ian"

Reply:
Honestly I am not sure if he should even be renamed, especially not with the `#1` part.

Reply:
Colonel Ian, _how didnt I think of that._ That should work better.